build_all/mac/release.sh

```diff
@@ -7,9 +7,15 @@ build_folder=$build_root"/c/cmake/iotsdk_mac"
 
 cd $build_root
 
-PYTHON_VERSION=2.7
+# build with Python 2 and Python 3 by default 
+PYTHON_VERSION=all
+
+# don't upload by default
 UPLOAD_PIP=0
 
+# processor architecture is any for Mac packages
+PLAT_ARCH="any"
```

Comment:
i'm still concerned by this. i'm afraid this wheel is going to be installable on systems that aren't macos and aren't covered by our other wheels. 
[This page](https://github.com/MacPython/wiki/wiki/Spinning-wheels) which is linked [from the official python docs](https://packaging.python.org/guides/packaging-binary-extensions/#binary-extensions-for-macos) suggests the platform tag to be macosx_<sdk_version>_intel (the page says it can be obtained by running`python -c "import distutils.util; print(distutils.util.get_platform())"`)

Reply:
I don't agree but respect your design choice. I will update the PR.